cmds/cmd_moderation.py

```diff
@@ -478,23 +493,36 @@ async def grab_guild_message(message, args, client, **kwargs):
 
     if not discord_message:
         await message.channel.send("Invalid MessageID")
+        return
 
-    if kwargs["main_version"] != "1.1.0 'LeXdPyK'":
-        files = grab_files(discord_message.guild.id, discord_message.id, kwargs["kernel_ramfs"])
+    # Generate replies
+    jump = f"\n\n[(Link)]({discord_message.jump_url})"
+    if (r := discord_message.reference) and (rr := r.resolved):
+        reply_contents = "> {} {}".format(rr.author.mention, rr.content.replace("\n", " "))[:511] + "\n"
```

Comment:
this should apply a `...` instead of brutally cutting it off and not giving any indication
same for the message content